person.rb

```diff
@@ -0,0 +1,40 @@
+require_relative './student'
+require_relative './teacher'
+
+class Person
+  extend Student
+  extend Teacher
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
+
+  def get
+    {
+      'id' => @id,
+      'name' => @name,
+      'age' => @age
+    }
+  end
+
+  def set(name, age)
+    @name = name
+    @age = age
+  end
```

Comment:
- [ ] You don't need to define getters and setters here since the attr_reader and attr_accessor are doing the exact same thing for you. Kindly remove this piece of code as it is redundant.